### PR TITLE
508 : sonar qube critical issue fixes

### PR DESCRIPTION
**Pull request by kanwarpreet25**
Pull request title

- Sonar Queb Issue :  Move constants to a class or enum.
- https://github.com/iluwatar/java-design-patterns/issues/508

Pull request description

- Fixes Sonar Qube Critical Issue : Move constants to a class or enum.

### CODE REVIEW

**Comment by kanwarpreet25** [comment]
@iluwatar  Can you assign someone to review this and merge

**Comment by iluwatar** [comment]
Thanks @kanwarpreet25 :+1:

File: double-dispatch/src/main/java/com/iluwatar/doubledispatch/SpaceStationMir.java [hunk 1]
```diff
@@ -45,29 +47,29 @@ public void collision(GameObject gameObject) {
...
   public void collisionResolve(SpaceStationMir mir) {
-    LOGGER.info("{} hits {}. {} is damaged!", mir.getClass().getSimpleName(),
+    LOGGER.info(AppConstants.HITS," {} is damaged!", mir.getClass().getSimpleName(),
         this.getClass().getSimpleName(), this.getClass().getSimpleName());
     setDamaged(true);
   }
 
   @Override
   public void collisionResolve(SpaceStationIss iss) {
-    LOGGER.info("{} hits {}. {} is damaged!", iss.getClass().getSimpleName(),
```
Comment (mookkiah) [inline double-dispatch/src/main/java/com/iluwatar/doubledispatch/SpaceStationMir.java:70]: I noticed this change resulted into more arguments than the placeholder. This resulting log statement as below
```
04:54:48.252 [main] INFO com.iluwatar.doubledispatch.SpaceStationMir -  {} is damaged! {} is set on fire! hits FlamingAsteroid.
04:54:48.278 [main] INFO com.iluwatar.doubledispatch.SpaceStationMir -  {} is damaged! {} is set on fire! hits FlamingAsteroid.

```

Comment (iluwatar) [reply double-dispatch/src/main/java/com/iluwatar/doubledispatch/SpaceStationMir.java:70]: Fixed in commit https://github.com/iluwatar/java-design-patterns/commit/195a735814b0b59a43998d9b0bba051d4b43ebcb

**Pull request merged**